content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/js/image.js

```diff
@@ -232,7 +232,7 @@
             }
             var url = that._properties.src.replace(SRC_URI_TEMPLATE_WIDTH_VAR, replacement);
 
-            if (that._elements.image.getAttribute("src") !== url) {
+            if (that._elements.image.getAttribute("src") === EMPTY_PIXEL) {
```

Comment:
We should also add a null check here, i guess, for the first load.
If you just load any AEM Page, containing an image component, in Authoring mode, you will notice that the image is not loaded without the null check.

```suggestion
            var imgSrcAttribute = that._elements.image.getAttribute("src");
            if (imgSrcAttribute === null || imgSrcAttribute === EMPTY_PIXEL) {
```